pycozmo/audio.py

```diff
@@ -24,19 +24,21 @@ def __init__(self, conn: ClientConnection):
         self.stream = []
         self._stop = False
         self.conn = conn
-        self.thread = Thread()
+        self.thread = None
         self.lock = Lock()
         self.audio_stream = []
 
     def start_stream(self):
         self._stop = False
-        if not self.thread.is_alive():
+        if not self.thread:
             self.thread = Thread(target=self.run, name=__class__.__name__)
             self.thread.start()
 
     def stop(self) -> None:
         self._stop = True
-        self.thread.join()
```

Comment:
Previously, if audio `play()` had never been called, this line was raising an exception as a thread that has not be started cannot be joined.